wasmCloud/wasmCloud
# fix(1365): CPU bound instances don't spread on all CPU cores

## Feature or Problem
Fix the cpu spread on all cores.
For me in my mind:
The change for the actor is to put in a spawn the wasmtime module clone, so the clone can happen on any core
The change on the rpc bus is to handle each nats event in a separate spawn in order to spread on all core

## Related Issues
 https://github.com/wasmCloud/wasmCloud/issues/1365

## Release Information
next

## Consumer Impact
Instances will use 100% of cpu

## Testing
none

### Unit Test(s)
none

### Acceptance or Integration
none

### Manual Verification
1/   set the wasmcloud RPC timeout high enough to actually wait for the response :
      export WASMCLOUD_RPC_TIMEOUT_MS=10000
2/   start wasmcloud 0.81.0
      wash up
3/   deploy the wadm file
      wash app deploy hello-world/wadm.yaml
4/   bench with any kind of app
      observe the cpu core loads
